bin/emqx

```diff
@@ -85,62 +85,59 @@ usage() {
     stop)
         echo "Stop the running EMQ X program"
         ;;
-    restart|reboot)
-        echo "Restart $EMQX_DESCRIPTION"
+    console)
+        echo "Boot up EMQ X service in an interactive Erlang or Elixir shell"
+        echo "This command needs a tty"
+        ;;
+    console_clean)
+        echo "This command does NOT boot up the EMQ X service"
+        echo "It only starts an interactive Erlang or Elixir console with all the"
+        echo "EMQ X code available"
+        ;;
+    foreground)
+        echo "Start EMQ X in foreground mode without an interactive shell"
         ;;
     pid)
-        echo "Print out $EMQX_DESCRIPTION process identifier"
+        echo "Print out EMQ X process identifier"
         ;;
     ping)
-        echo "Check if the $EMQX_DESCRIPTION node is up and running"
+        echo "Check if the EMQ X node is up and running"
         echo "This command exit with 0 silently if node is running"
         ;;
     escript)
         echo "Execute a escript using the Erlang runtime from EMQ X package installation"
         echo "For example $REL_NAME escript /path/to/my/escript my_arg1 my_arg2"
         ;;
     attach)
-        echo "This command is applicable when $EMQX_DESCRIPTION is started in daemon"
+        echo "This command is applicable when EMQ X is started in daemon"
         echo "mode. it attaches the current shell to EMQ X's control console"
         echo "through a named pipe"
         echo "WARNING: try to use the safer alternative, remote_console command."
         ;;
     remote_console)
-        echo "Start a dummy Erlang node and hidden-connect $EMQX_DESCRIPTION to"
-        echo "with an interactive Erlang shell"
-        ;;
-    console)
-        echo "Boot up $EMQX_DESCRIPTION service in an interactive Erlang shell"
-        echo "This command is useful for troubleshooting"
-        ;;
-    console_clean)
-        echo "This command does NOT boot up the $EMQX_DESCRIPTION service"
-        echo "It only starts an interactive Erlang console with all the"
-        echo "EMQ X code available"
-        ;;
-    foreground)
-        echo "Start $EMQX_DESCRIPTION in foreground mode"
+        echo "Start a dummy Erlang or Elixir node and hidden-connect EMQ X to"
+        echo "with an interactive Erlang or Elixir shell"
         ;;
     ertspath)
         echo "Print path to Erlang runtime dir"
         ;;
     rpc)
         echo "Usge $REL_NAME rpc MODULE FUNCTION [ARGS, ...]"
-        echo "Connect to the $EMQX_DESCRIPTION node and make an Erlang RPC"
-        echo "The result of the RPC call must be 'ok'"
-        echo "This command blocks for at most 60 seconds in case the node"
-        echo "does not reply the call in time"
+        echo "Connect to the EMQ X node and make an Erlang RPC"
+        echo "This command blocks for at most 60 seconds."
+        echo "It exits with non-zero result in case of failure"
```

Comment:
By failure it means exception, or failure to connect to a node (e.g. when it's not running)?

Reply:
rephrased